docs/application/native/guides/ui/efl/efl-util.md

```diff
@@ -259,8 +259,11 @@ To generate key input events:
        /* Create an input device with the default name: "Input Generator" */
        inputgen = efl_util_input_initialize_generator(EFL_UTIL_INPUT_DEVTYPE_KEYBOARD);
        /*
-          Since Tizen 4.0, you can create an input generator with a given name (in this example, "Local Device"):
+          Since Tizen 4.0, you can create an input device with a given name (in this example, "Local Device"):
           inputgen = efl_util_input_initialize_generator_with_name(EFL_UTIL_INPUT_DEVTYPE_KEYBOARD, "Local_Device");
+
+          Since Tizen 5.0, you can create an input device synchronously with a given name (device_name can be NULL):
```

Comment:
Please update this sentence also as per the suggestions given in the comments.

Reply:
I don't understand what you said, please said more detail.
I alread fixed all sentense to input device not input generator.

Reply:
Hello,
Please consider the first part of the comment as well.

Suggestion:
From, Tizen 5.0, you can create an input generator synchronously with a given name (device_name can be NULL):